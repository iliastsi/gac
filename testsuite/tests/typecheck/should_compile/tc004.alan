unusedFun() : proc
  foo1() : proc
  {}
  foo2() : proc
    foo3() : proc
    {}
  { foo1(); }
{}
