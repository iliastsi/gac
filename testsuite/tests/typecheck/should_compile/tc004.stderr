tc004.alan:4:3-6: Warning: Defined but not used: `foo2'

tc004.alan:5:5-8: Warning: Defined but not used: `foo3'

